Source/WebKit/Shared/ShareableBitmap.h

```diff
@@ -119,6 +119,8 @@ class ShareableBitmapHandle  {
 
 class ShareableBitmap : public ThreadSafeRefCounted<ShareableBitmap> {
 public:
+    using Handle = ShareableBitmapHandle;
```

Comment:
Would be nice to update the `ShareableBitmap` implementation to use `Handle` instead of `ShareableBitmapHandle` now.

Reply:
Doh! Thought I did that! Will fix.

Reply:
Fixed

Reply:
Ok now for sure I've fixed it @cdumez